### PR TITLE
PANTHER IDF fix

### PR DESCRIPTION
**Pull request by gvardany**
**Description of work.**
Corrects the IDF of PANTHER following the calibration measurements during the hot commissioning.
Confirmed by the instrument scientists.
The PR of the generator can be found [here](https://github.com/mantidproject/mantidgeometry/pull/174)

**To test:**
1) `rm -rf ~/.mantid/instrument`
2) Run LoadEmptyInstrument with PANTHER as the name.
3) Open Instrument Viewer, go to pick tab.
4) Mouse hover on 80th pixel of any of the tubes. Make sure that for those `y=0`, `r=2.5`.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by darominski — DISMISSED** [review]
Everything looks good; the instrument viewer works as described and the modified tests work well as well.

File: docs/source/release/v6.0.0/direct_geometry.rst [hunk 1]
```diff
@@ -5,6 +5,12 @@ Direct Geometry Changes
 .. contents:: Table of Contents
    :local:
 
+Improvements
+------------
+
+- The instrument geometry of PANTHER has been corrected according to the findins during the hot commissioning.
```
Comment (darominski) [inline docs/source/release/v6.0.0/direct_geometry.rst:11]: There's a missing 'g' in 'findings'

**Review by darominski — COMMENTED** [review]
Just a typo in the doc found by @gui-co

File: docs/source/release/v6.0.0/direct_geometry.rst (hunk 1 above)
Comment (gvardany) [reply docs/source/release/v6.0.0/direct_geometry.rst:11]:
```suggestion
- The instrument geometry of PANTHER has been corrected according to the findings during the hot commissioning.
```

**Comment by gvardany** [comment]
I fixed the type, can you approve again?

**Review by darominski — APPROVED** [review]

**Pull request merged**